ci/cloudbuild/triggers/gcc-54-pr.yaml

```diff
@@ -5,7 +5,9 @@ github:
   pullRequest:
     branch: ^master$
     commentControl: COMMENTS_ENABLED_FOR_EXTERNAL_CONTRIBUTORS_ONLY
-name: gcc-54
+name: gcc-54-pr
 substitutions:
   _BUILD_NAME: gcc-54
   _DISTRO: fedora
```

Comment:
I did not notice this before, but does fedora have GCC 5.4?

Reply:
Nope. That's a bug that I introduced in https://github.com/googleapis/google-cloud-cpp/pull/6135. I'll fix that in a followup PR.